File: app/src/main/java/android/mb00/codehub/api/model/PullRequest.java
package android.mb00.codehub.api.model;

import com.google.gson.annotations.SerializedName;


public class PullRequest {

    @SerializedName("state")
    private String state;
    @SerializedName("title")
    private String title;
    @SerializedName("number")
    private String number;
    @SerializedName("created_at")
    private String creationDate;
    @SerializedName("user")
    private User user;

    public String getState() {
        return state;
    }

    public String getTitle() {
        return title;
    }

    public String getNumber() {
        return number;
    }

    public String getCreationDate() {
        return creationDate;
    }

    public User getUser() {
        return user;
    }

}

File: app/src/main/java/mb00/android/codehub/api/model/PullRequest.kt
package mb00.android.codehub.api.model

import com.google.gson.annotations.SerializedName


data class PullRequest(
        @SerializedName("state") var state: String?,
        @SerializedName("title") var title: String?,
        @SerializedName("number") var number: String?,
        @SerializedName("created_at") var creationDate: String?,
        @SerializedName("user") var user: User?
)
